llama_stack/providers/remote/inference/ollama/ollama.py

```diff
@@ -94,10 +99,11 @@ async def completion(
         response_format: Optional[ResponseFormat] = None,
         stream: Optional[bool] = False,
         logprobs: Optional[LogProbConfig] = None,
-    ) -> AsyncGenerator:
+    ) -> CompletionResponse | AsyncGenerator[CompletionResponseStreamChunk, None]:
+        assert self.model_store is not None
         if sampling_params is None:
             sampling_params = SamplingParams()
-        model = await self.model_store.get_model(model_id)
```

Comment:
this one is suspicious. get_model as declared elsewhere seems to return Model, not AsyncGenerator. But on the other hand, not sure how it works right now. Need to take a closer look before merge.